tests/jsx-split-attrs/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -122,8 +122,7 @@ long_open_long_children = (
         colour="blue"
         size="large"
         submitLabel="Sign in with Google"
-      />
-      d
+      />d
```

Comment:
This really seems wrong to me... is there any way to not use this algorithm if one of the elements has broken?